components/camel-graphql/src/main/java/org/apache/camel/component/graphql/GraphqlEndpoint.java

```diff
@@ -65,6 +65,8 @@
     @UriParam(label = "security", secret = true)
     private String password;
     @UriParam
+    private String jwtAuthorizationType;
```

Comment:
Can you set label = "security", and default = "Bearer" in the uri param annotation

Reply:
Sure thing, added those now